Examples/AppDelegate.m

```diff
@@ -7,18 +7,34 @@
 //
 
 #import "AppDelegate.h"
+@import Mapbox;
 
 @interface AppDelegate ()
 
 @end
 @implementation AppDelegate
+NSString * const MBXMapboxAccessTokenDefaultsKey = @"MBXMapboxAccessToken";
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     // Speed-up Core Animation-based animations in testing scenarios.
     if ([[NSProcessInfo processInfo].arguments containsObject:@"useFastAnimations"]) {
         self.window.layer.speed = 100;
     }
 
+    if ( ! [MGLAccountManager accessToken]) {
```

Comment:
This project expects the developer to always provide a token, which is then copied into `Info.plist` by [a build script](https://github.com/mapbox/ios-sdk-examples/blob/9298086b29b80f49c92c101215a242c0f5659c25/Examples/insert-mapbox-token.sh). If that build script doesn’t find a token in the `mapbox_access_token` file, the build will fail.

... which is to say, this check in the app delegate will never fail and shouldn’t be necessary.

Reply:
> this check in the app delegate will never fail

Actually, I suppose this statement isn’t strictly true: this project relies on the the token being in `Info.plist` when the map loads (and never explicitly sets a token), so this check _would_ succeed in setting a token that was included in the project’s environmental variables (but it doesn’t have to).